### PR TITLE
Update the documentation for OAP 1.1.0

### PR DESCRIPTION
**Pull request by HongW2019**

### CODE REVIEW

**Comment by HongW2019** [comment]
@yao531441 Please take a review and help merge it.

File: docs/docs/OAP-Developer-Guide.md [hunk 1]
```diff
@@ -9,31 +9,15 @@ You can get more detailed information from OAP each module below.
 * [PMem Shuffle](https://github.com/oap-project/pmem-shuffle#5-install-dependencies-for-shuffle-remote-pmem-extension)
 * [Remote Shuffle](https://github.com/oap-project/remote-shuffle)
 * [OAP MLlib](https://github.com/oap-project/oap-mllib)
-* [Arrow Data Source](https://github.com/oap-project/arrow-data-source)
+* [Arrow Data Source](https://github.com/oap-project/native-sql-engine/tree/branch-1.1-spark-3.x/arrow-data-source)
 * [Native SQL Engine](https://github.com/oap-project/native-sql-engine)
```
Comment (yao531441) [inline docs/docs/OAP-Developer-Guide.md:13]: We can remove Arrow Data Source

File: docs/docs/OAP-Developer-Guide.md [hunk 2]
```diff
@@ -9,31 +9,15 @@ You can get more detailed information from OAP each module below.
...
-
-- **Requirements for Shuffle Remote PMem Extension**  
-If enable Shuffle Remote PMem extension with RDMA, you can refer to [PMem Shuffle](https://github.com/oap-project/pmem-shuffle) to configure and validate RDMA in advance.
-
-We provide scripts below to help automatically install dependencies above **except RDMA**, need change to **root** account, run:
+We provide scripts to help automatically install dependencies listed below **except RDMA**, need change to **root** user, run:
 
 ```
-# git clone -b <version> https://github.com/oap-project/oap-tools.git
+# git clone -b <tag-version> https://github.com/oap-project/oap-tools.git
```
Comment (yao531441) [inline docs/docs/OAP-Developer-Guide.md:20]: Add notes to explain branch-version.
# branch oap-1.1 corresponding to OAP 1.1
tag-version => branch-version

File: docs/docs/OAP-Developer-Guide.md [hunk 3]
```diff
@@ -50,6 +34,20 @@ Run the following command to automatically install specific dependency such as M
 # sh dev/scripts/prepare_oap_env.sh --prepare_maven
 ```
 
+OAP is built with [Apache Maven](http://maven.apache.org/) and Oracle Java 8, and mainly required tools to install on your cluster are listed below.
+
```
Comment (yao531441) [inline docs/docs/OAP-Developer-Guide.md:38]: Add: You can ignore it if you execute the script above

File: docs/docs/OAP-Installation-Guide.md [hunk 4]
```diff
@@ -19,9 +19,19 @@ $ wget -c https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
 $ chmod +x Miniconda2-latest-Linux-x86_64.sh 
 $ bash Miniconda2-latest-Linux-x86_64.sh 
 ```
-For changes to take effect, close and re-open your current shell. To test your installation,  run the command `conda list` in your terminal window. A list of installed packages appears if it has been installed correctly.
+For changes to take effect, ***close and re-open***your current shell.
```
Comment (yao531441) [inline docs/docs/OAP-Installation-Guide.md:22]: ***close and re-open***your miss a space

File: docs/docs/OAP-Developer-Guide.md (hunk 1 above)
Comment (HongW2019) [reply docs/docs/OAP-Developer-Guide.md:13]: Done

**Comment by HongW2019** [comment]
@yao531441 Please take a review and help merge it.

**Review by yao531441 — APPROVED** [review]
LGTM!

**Pull request merged**